deployments/deployments.yml

```diff
@@ -42,7 +42,6 @@ eip155:84532:
   deployment id: MVP
   env: test
   name: BASE_SEPOLIA
-  env: test
```

Comment:
Should we remove base sepolia from deployments altogether? Since we do not use testnets anymore

Reply:
yes please